### PR TITLE
RFR: Improvements to kubernetes pack!

### PR DESCRIPTION
**Pull request by lakshmi-kannan**
## TODO:

* - [X] Cleanup code
* - [X] Unit tests

### CODE REVIEW

**Comment by lakshmi-kannan** [comment]
I added some unit tests as well for the sensor. I could write more tests and honestly we should but I'll stop here in the interest of time. Would be nice to write tests for the action as well.

File: packs/kubernetes/sensors/third_party_resource.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
```
Comment (manasdk) [inline packs/kubernetes/sensors/third_party_resource.py]: remove

Comment (lakshmi-kannan) [inline packs/kubernetes/sensors/third_party_resource.py]: bleh. missed it.

File: packs/kubernetes/tests/test_third_party_resource.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
+from st2tests.base import BaseSensorTestCase
```
Comment (manasdk) [inline packs/kubernetes/tests/test_third_party_resource.py]: Do we add license in all code files?

**Comment by manasdk** [comment]
+1 ... asuumin travis is happy ofcourse :)

File: packs/kubernetes/tests/test_third_party_resource.py (hunk 2 above)
Comment (lakshmi-kannan) [inline packs/kubernetes/tests/test_third_party_resource.py]: It's a mixed bag unfortunately in st2contrib.

**Comment by lakshmi-kannan** [comment]
Yeah, these travis tests aren't working correctly :P. I renamed the metadata file. I'll merge.

**Pull request merged**